packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1803,6 +1803,26 @@ For brevity, the following response doesn't include all available properties.
         "description": "Configuration properties specific to Splunk Cloud",
         "type": "object",
         "properties": {
+          "edition": {
+            "title": "Splunk Edition",
+            "description": "Select a Splunk Edition.",
```

Comment:
```suggestion
            "description": "Select a Splunk Cloud instance edition.",
```

Reply:
This description defines labels in the UI, I'd leave it as is for now